index.rst

```diff
@@ -1,13 +1,14 @@
-SimpleITK Sphinx Documentation
-==============================
+Welcome to SimpleITK 
+=====================
 
-SimpleITK is a simplified, open source, interface to the National Library
-of Medicine's Insight Segmentation and Registration Toolkit (ITK), a C++
-open source image analysis toolkit which is widely used in academia and
-industry. SimpleITK is available for eight programming languages including
-C++, Python, R, Java, C#, Lua, Ruby, and TCL. Binary distributions of
-SimpleITK are currently available for all three major operating systems
-(Linux, OS X, and Windows).
+SimpleITK is an image-analysis toolkit, providing a simplified programming interface to the algorithms and data structures of the Insight Segmentation and Registration Toolkit. It supports image operations on 2D, 3D and 4D multi-channel images. The toolkit is available in eight programming languages including Python, R, Java, C#, C++, Lua, Ruby, and TCL.
+
+SimpleITK includes several hundred classes providing a broad set of tools required for image analysis. The toolkit provides extensive support for image input and output, supporting more than 20 image file formats. A large number of filtering operations are also available, from basic image analysis operations such as edge detection, binary and grayscale morphological operations, and distance transformations to MRI bias field correction. Additionally, a variety of traditional operators used for image segmentation are available including region growing, connected component labelling, and the watershed transformation. Finally, the toolkit includes a robust registration framework, readily enabling one to align intra and inter modality images, supporting both rigid and non-rigid transformation models.
```

Comment:
'labelling' is the British spelling.  The rest of SimpleITK uses 'labeling', the American spelling.  I'd suggest going with the American spelling, mainly to be consist.